content/en/logs/guide/azure-automated-log-forwarding.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Azure Automated Log Forwarding Setup
+private: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/monitoring-azure-platform-logs/"
+  tag: "Blog"
+  text: "Best practices for monitoring Microsoft Azure platform logs"
+---
+
+## Overview
+
+<div class="alert alert-info">Azure automated log forwarding is in Preview.</div>
+
+Use this guide to automate your Azure log forwarding setup with an Azure Resource Manager (ARM) template. 
+
+The ARM template deploys resources from a series of Azure services (storage accounts and function apps) into your subscriptions that collect and forward logs to Datadog. These services automatically scale up or down to match log volume. Scaling is manged by a control plane, which is a set of function apps deployed to a subscription and region of your choice. Storage accounts and function apps are deployed in each of the subscriptions forwarding logs to Datadog.
+
+**All sites**: All [Datadog sites][4] can use the steps on this page to complete setup of automated log forwarding.
+
+## Setup
+
+Begin by opening the [Automated Log Forwarding ARM template][1]. The sections below provide instructions for completing each page of the template.
+
+### Basics
```

Comment:
The only small nit I have about this section is that it's a bit hard to scan. Sometimes, if a user is struggling with what to put in a field, they might `ctrl+f` for the field name to look for a description, or try to scan through the list of fields themselves to find it. Others may have visual impairments that prevent them from seeing the field names in your screenshot. It's not a huge deal to keep this as-is, especially if you're busy; but if you do have time, it might be helpful to use exact field names to help users find them more easily.

Reply:
Thanks for that feedback; updated in [8af402b](https://github.com/DataDog/documentation/pull/27355/commits/8af402b85d8005b51e72c75cdb92b76171068931)